commit 585d6c46e5b9659d062a3609c0bf8d290b692b12 (refs/changes/41/541/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-28T17:43:06-07:00 (3 years ago)
    
    PUBAPI-1326 Update sdcnode version for OpenSSL 1.0.2j/1.0.1u vulnerability fixes
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index 571a631..753d0fa 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ CLEAN_FILES	+= node_modules cscope.files
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v0.10.46
+NODE_PREBUILT_VERSION=v0.10.47
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
 	NODE_PREBUILT_TAG=zone
